experimental/packages/opentelemetry-instrumentation-grpc/src/enums/AttributeNames.ts

```diff
@@ -19,7 +19,9 @@
  */
 export enum AttributeNames {
   GRPC_KIND = 'grpc.kind', // SERVER or CLIENT
-  GRPC_METHOD = 'grpc.method',
+  GRPC_METHOD = 'rpc.method',
```

Comment:
I think the `RPC_METHOD` of `@opentelemetry/semantic-convention` could be used for this?

Reply:
See: https://github.com/open-telemetry/opentelemetry-js/blob/747c404fb1295d54eb628a434ebf782d2f180bfb/packages/opentelemetry-semantic-conventions/src/trace/SemanticAttributes.ts#L676

Reply:
@weyert, resolved
There were some [issues](https://github.com/microsoft/TypeScript/issues/40793) because of Typescript
It forced me to make enum as Readonly constant